src/bundle.ts

```diff
@@ -0,0 +1,20 @@
+// allows us to specify --noEmitHelpers within our tsconfig.json
+// this skips emitting helpers in every file, we just load them once here
+import 'ts-helpers';
+
+// Export all classes (named and default) that you want to be publicly available
+// in the browser as named exports here.
+// Interfaces should be ignored, as they don't export any code.
+import { default as _export } from './lib/EventDispatcher';
+
+export { default as IEventDispatcher } from './lib/IEventDispatcher';
+export { default as IEvent } from './lib/IEvent';
+export { default as EventPhase } from './lib/EventPhase';
+export { default as EventListenerData } from './lib/EventListenerData';
+export { default as CallListenerResult } from './lib/CallListenerResult';
+export { default as AbstractEvent } from './lib/AbstractEvent';
+import './lib/event/CommonEvent';
```

Comment:
Should CommonEvent be part of the lib, as it doesn't even get exported for public usage.

Reply:
The import itself doesn't make much sense indeed

Reply:
I'll remove the CommonEvent entirely and update the README.md example to be sufficient!

Reply:
Not sure if I agree with removing the file itself. It's added in the package for convenience, because those type of events are used most of the time. You can import them yourself from the `seng-event/lib` folder if you need it, and it will only get included in your project output when you do that.

A lot of packages include a set of related useful files/plugins/adaptors with their main package.

Reply:
Then I would like to purpose to add DataEvent as well.